internal/controller/metrics.go

```diff
@@ -61,6 +61,13 @@ var (
 		},
 		[]string{"node_disruption_name", "node_name"},
 	)
+	NodeDisruptionType = promauto.With(metrics.Registry).NewGaugeVec(
```

Comment:
Actually, I think it's better to add `type` on the labels list of all the node disruption metrics (and remove the node you added):
e.g.:
```
	NodeDisruptionImpactedNodes = promauto.With(metrics.Registry).NewGaugeVec(
		prometheus.GaugeOpts{
			Name: METIC_PREFIX + "node_disruption_impacted_node",
			Help: "high cardinality: create a metric for each node impacted by a given node disruption",
		},
		[]string{"node_disruption_name", "node_name", "type"},  <---- here
	)
```

Reply:
Done ✅